content/en/docs/concepts/storage/volumes.md

```diff
@@ -996,18 +996,22 @@ For more information, see the [vSphere volume](https://github.com/kubernetes/exa
 
 {{< feature-state for_k8s_version="v1.19" state="beta" >}}
 
-The `CSIMigration` feature for `vsphereVolume`, when enabled, redirects all plugin operations
-from the existing in-tree plugin to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this feature, the
+The `CSIMigration` feature for `vsphereVolume`,  is enabled by default as of Kubernetes v1.25.
+All plugin operations from the in-tree `vspherevolume` will be redirected to the `csi.vsphere.vmware.com` {{< glossary_tooltip text="CSI" term_id="csi" >}} driver. 
+
 [vSphere CSI driver](https://github.com/kubernetes-sigs/vsphere-csi-driver)
-must be installed on the cluster and the `CSIMigration` and `CSIMigrationvSphere`
-[feature gates](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled.
-You can find additional advice on how to migrate in VMware's
+must be installed on the cluster. You can find additional advice on how to migrate in VMware's
```

Comment:
The sentence on migration doesn't sound right because we're not specifying what we're migrating to or what we're migrating from. 

```suggestion
must be installed on the cluster. You can find additional advice on how to migrate in VMware's
```

Reply:
changed it to 

> You can find additional advice on how to migrate in-tree `vsphereVolume` in VMware's
documentation page`